src/processor/StructureDefinitionProcessor.ts

```diff
@@ -21,6 +21,7 @@ import {
 } from '../extractor';
 import { ProcessableElementDefinition } from '.';
 import { getAncestorSliceDefinition } from '../utils';
+import { switchQuantityRules } from './common';
```

Comment:
Minor thing, but if you add:
```javascript
export * from './common';
```
to `src/processor/index.ts`, then on line 22 here, you can do:
```javascript
import { ProcessableElementDefinition, switchQuantityRules } from '.';
```